.npmrc

```diff
@@ -0,0 +1,6 @@
+strict-peer-dependencies=false
+auto-install-peers=true
+resolution-mode=highest
+engine-strict=true
+use-node-version=20.10.0
+prefer-workspace-packages=true 
```

Comment:
@samueljd , can you add an empty line at the of the files, helps when moving across OSs and conflict checking within git

Reply:
Interesting conversation about the newlines: https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log